core/src/main/java/org/neo4j/ogm/metadata/FieldInfo.java

```diff
@@ -65,6 +65,7 @@
     private final boolean isArray;
     private final boolean isSupportedNativeType;
     private final ClassInfo containingClassInfo;
+    private volatile Optional<String> relationship;
```

Comment:
Please rename `relationshipType`, move below `convertedType` and a similar comment.

Reply:
Hope the comment is ok ;)